docs/getting-started-guides/ubuntu/troubleshooting.md

```diff
@@ -105,6 +105,102 @@ charm unit data, etc. Additional application-specific information may be
 included as well.
 
 ## Common Problems
+### Load Balancer interfering with Helm
+
+This section assumes you have a working deployment of Kubernetes via Juju using a Load Balancer for the API, and that you are using Helm to deploy charts. 
+
+To deploy Helm you will have run: 
+
+```
+helm init
+$HELM_HOME has been configured at /home/ubuntu/.helm
+Tiller (the helm server side component) has been installed into your Kubernetes Cluster.
+Happy Helming!
+```
+
+Then when using helm you may see one of the following errors: 
+
+* Helm doesn't get the version from the Tiller server
+
+```
+helm version
+Client: &version.Version{SemVer:"v2.1.3", GitCommit:"5cbc48fb305ca4bf68c26eb8d2a7eb363227e973", GitTreeState:"clean"}
+Error: cannot connect to Tiller
+```
+
+* Helm cannot install your chart
+
+```
+helm install <chart> --debug
+Error: forwarding ports: error upgrading connection: Upgrade request required
+```
+
+This is caused by the API load balancer not forwarding ports in the context of the helm client-server relationship. To deploy using helm, you will need to follow these steps: 
```

Comment:
We really need to capture this as a bug and make it work ootb. This is a great work around in the interrim. Thank you for capturing these steps and getting it placed in the docs 👍